openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -67,6 +68,9 @@ def send_credit_notifications(username, course_key):
         # strip enclosing angle brackets from 'logo_image' cache 'Content-ID'
         logo_image_id = logo_image.get('Content-ID', '')[1:-1]
 
+    providers = get_credit_providers(user, course_key)
+    providers = ", ".join(providers) if providers else None
```

Comment:
It looks like this providers list is going to be used in sentences displayed to the user. At the moment, this is just going to return a comma-separated list of providers. It would be better if the number of providers available influenced the structure of the string:
```
One provider => "A"
Two providers => "A and B"
Three or more providers => "A, B, ..., and N"
```
This is also a tricky i18n problem, but fortunately one that we've solved before: https://github.com/edx/edx-platform/commit/1d1baa50098c1108a4ae7e0c2fc345fbc8dd8a3c#diff-d8e9b05acd3b7dba84709ae01b99053cR858